evm/utils/env.py

```diff
@@ -0,0 +1,301 @@
+    a ``ValueError``)
+    :type default: bool
+    """
+    value = get_env_value(name, default=default, required=required)
+    if value is empty:
+        value = ''
+    return value
+
+
+def env_list(name, separator=',', required=False, default=empty):
+    """Pulls an environment variable out of the environment, splitting it on a
+    separator, and returning it as a list. Extra whitespace on the list values
+    is stripped. List values that evaluate as falsy are removed. If not present
+    and no default specified, an empty list is returned.
+
+    :param name: The name of the environment variable be pulled
+    :type name: str
+
+    :param separator: The separator that the string should be split on.
+    :type separator: str
+
+    :param required: Whether the environment variable is required. If ``True``
+    and the variable is not present, a ``KeyError`` is raised.
+    :type required: bool
+
+    :param default: The value to return if the environment variable is not
+    present. (Providing a default alongside setting ``required=True`` will raise
+    a ``ValueError``)
+    :type default: bool
+    """
+    value = get_env_value(name, required=required, default=default)
+    if value is empty:
+        return []
+    # wrapped in list to force evaluation in python 3
+    return list(filter(bool, [v.strip() for v in value.split(separator)]))
+
+
+def env_timestamp(name, required=False, default=empty):
+    """Pulls an environment variable out of the environment and parses it to a
+    ``datetime.datetime`` object. The environment variable is expected to be a
+    timestamp in the form of a float.
+
+    If the name is not present in the environment and no default is specified
+    then a ``ValueError`` will be raised.
+
+    :param name: The name of the environment variable be pulled
+    :type name: str
+
+    :param required: Whether the environment variable is required. If ``True``
+    and the variable is not present, a ``KeyError`` is raised.
+    :type required: bool
+
+    :param default: The value to return if the environment variable is not
+    present. (Providing a default alongside setting ``required=True`` will raise
+    a ``ValueError``)
+    :type default: bool
+    """
+    if required and default is not empty:
+        raise ValueError("Using `default` with `required=True` is invalid")
+
+    value = get_env_value(name, required=required, default=empty)
+    # change datetime.datetime to time, return time.struct_time type
+    if default is not empty and value is empty:
+        return default
+    if value is empty:
+        raise ValueError(
+            "`env_timestamp` requires either a default value to be specified, "
+            "or for the variable to be present in the environment"
+        )
+
+    timestamp = float(value)
+    return datetime.datetime.fromtimestamp(timestamp)
+
+
+def env_iso8601(name, required=False, default=empty):
```

Comment:
I'm skeptical that we'll use this utility.  What do you think about removing it (and the related tests).

Reply:
I think it's fine to remove it. I cannot figure out a scenario to make use of it.